courses/tests/test_signals.py

```diff
@@ -0,0 +1,40 @@
+"""
+Signal tests
+"""
+# pylint: disable=no-self-use
+import mock
+
+from django.test import TestCase
+
+from courses.factories import CourseFactory, ModuleFactory
+
+
+class PublishOnUpdateTests(TestCase):
+    """
+    Tests for webhook posting
+    """
+    def test_course_save_publishes(self):
+        """
+        When course is saved, it should trigger webhook.
+        """
+        c = CourseFactory.build()
```

Comment:
In general we should avoid single letter variables like these in favor of `course` and `module`, though it's not super important here due to the limited scope of the variables